gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/x509/X509Config.groovy

```diff
@@ -72,6 +79,17 @@ class X509Config {
     }
   }
 
+  @ConditionalOnBean(X509StandaloneAuthConfig)
+  @Bean
+  AuthenticationManager authenticationManagerBean() {
+    new AuthenticationManager() {
+      @Override
+      Authentication authenticate(Authentication authentication) throws AuthenticationException {
+        throw new UnsupportedOperationException("No authentication should be done with this AuthenticationManager");
```

Comment:
`authenticate` is never called here, correct? Seems very strange that this bean is required. Probably would be a good idea to get @cfieber or @ajordens since this use case is mostly theirs.